Cargo.toml

```diff
@@ -10,11 +10,12 @@ license = "MIT"
 include = ["src/**/*", "Cargo.*", "LICENSE", "README.md", "CHANGELOG.md", "!**/tests/*"]
 
 [features]
-default = ["tui-crossplatform"]
+default = ["tui-crossplatform", "trash-move"]
```

Comment:
Am I the only one with a smirk in the face when reading the feature name ;)?  It's very fitting though.